src/device.coffee

```diff
@@ -44,10 +44,19 @@ exports.getID = do ->
 exports.reboot = ->
 	request.postAsync(config.gosuperAddress + '/v1/reboot')
 
+exec = require('child_process')
```

Comment:
this should go to the top section, and better be `{ execSync } = require('child_process ')` (then just use this bare function)